strapi/strapi
# Fix 21368: fixed the pagination showing for less assets

What does it do?
Fixed the pagination showing for less number of Assets

Why is it needed?
To fix issue [21368](https://github.com/strapi/strapi/issues/21368): When the number of assets is lower than the amount that should be displayed, the pagination shouldn’t be visible

How to test it?

1. Open the Content Manager.
2. Select a collection type or single type that allows you to add an image.
3. Add an image; pagination should appear if the asset count exceeds the page size.

Related issue(s)/PR(s)
[21368](https://github.com/strapi/strapi/issues/21368)
